### ISSUE TITLE
Required Boost version seems to high

### ISSUE THREAD

**Issue by verri** [alandefreitas/clang-unformat#10]
First of all, congratulations for this great tool.

I was compiling in Ubuntu 22.04, which uses Boost 1.74 by default, and I realized that the current CMakeLists.txt requires boost 1.78.

I have changed the file to require version 1.74, and it compiled and executed fine.

**Comment by verri** [comment]
Also FetchContent is available only for CMake 3.11 or above.

**Comment by alandefreitas** [comment]
Yes. Updating the version requirements is always helpful. I didn't worry too much about looking for the minimal version that would work because this is only an application that can be compiled once so other libraries won't directly depend on it. Same for FetchContent, which could check the CMake version.

**Comment by royshil** [comment]
i was able to build with boost v1.74 and also removing boost-process from the required components

**Comment by alandefreitas** [comment]
Yes. That should work.
